controllers/account-setting/my-team/remove-access-by-agent-supervisor.controller.js

```diff
@@ -8,7 +8,15 @@ const removeMyTeamAccessRequestByAgentOrSupervisor = async (req, res, next) => {
 	try {
 		const { documentId, requestId } = req.body;
 		const user = req.user;
-		
+
+		if (user.role !== 'CRM_ADMIN' && user.role !== 'CRM_SUPERVISOR') {
```

Comment:
This is redundant and bad practise. use a middleware. Already implemented with routes in notifications or auth